### PR TITLE
Add data views units of measure docs

### PR DESCRIPTION
**Pull request by d-ng**

### CODE REVIEW

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+If the unit of measure is consistent across all groups, then that unit of measure is reported on the [field mapping](xref:DataViewsFieldMappings) itself. Otherwise, the field mapping's `Uom` property is null.
+
+Note that the [ids of the units of measure](xref:unitsOfMeasure) are shown in resolved resources.
+
+
+## Units of measure in data view
+Data views can be optionally configured to show units of measure within the data set or in labels. The [abbreviations of the units of measure](xref:unitsOfMeasure) are shown in the data view.
+
+### Show units of measure within the data set
+To show the units of measure within the data set, set the field's `IncludeUom` property to `true`. This apply to both data fields and group fields. In this case, the field will resolve an additional, "sister" field mapping whose label is suffixed with the string "Uom'.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:24]: Possessives:  change "...set the field's IncludeUom property to true." to "...set the IncludeUom property of the field to true."

Change "This apply..." to "...This applies..."

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+## Units of measure in data view
+Data views can be optionally configured to show units of measure within the data set or in labels. The [abbreviations of the units of measure](xref:unitsOfMeasure) are shown in the data view.
+
+### Show units of measure within the data set
+To show the units of measure within the data set, set the field's `IncludeUom` property to `true`. This apply to both data fields and group fields. In this case, the field will resolve an additional, "sister" field mapping whose label is suffixed with the string "Uom'. 
+
+#### Example
+This simple example show a data field's `IncludeUom` property is set to `true`.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:27]: Change "This simple example show a data field's IncludeUom property..." to "The following example shows the IncludeUom property of a field is set to true."

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+| Timestamp            | Field | Value | Uom |
+|----------------------|-------|-------|-----|
+| 2020-11-06T08:00:00Z | Value | 1.21  | kW  |
+
+
+### Show units of measure in field labels
+The field label token {Uom} can be used to add field-level unit of measure to field labels. {Uom} is included in the default field label. If the unit of measure is consistent across all groups, {Uom} is resolved to the field-level unit of measure and displayed in the field label.
+
+#### Example
+This simple example show how the field label token {Uom} of a data field is resolved.
```
Comment (osimarkrogers) [inline Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:69]: Change "This simple example show..." to "The following example shows..."

**Review by osimarkrogers — CHANGES_REQUESTED** [review]

File: Content_Portal/Documentation/DataViews/DefineaDataView/Field_Mappings.md [hunk 4]
```diff
@@ -58,9 +60,10 @@ Fields of kind `DataField` map to a [data field](xref:DataViewsFieldSets) on the
 | DataMapping Property | Details |
 |--|--|
 | TargetId | The data item id |
-| TargetMeasurementKey *(Coming Soon)* | The asset measurement's name. Only applicable to `PropertyId` fields and when the data field references an asset measurement. |
+| TargetMeasurementKey *(coming soon)* | The asset measurement's name. Only applicable to `PropertyId` fields and when the data field references an asset measurement. |
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Field_Mappings.md:63]: sure

File: Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md [hunk 5]
```diff
@@ -100,6 +100,7 @@ All sources except `FieldSource.NotApplicable` can be used as data fields. Field
 | Keys | String[] | Optional | [ ] | Used for sources `FieldSource.PropertyId`, `FieldSource.PropertyName`, `FieldSource.Metadata` and `FieldSource.Tags`, e.g. to map to specific data item properties by id. If more than one key is specified, they are matched as exclusive-or. A key has to be a non-null value.
 | MeasurementKeys *(Coming Soon)* | String[] | Optional | [ ] | Identifies the measurement name when referencing an asset measurement. `MeasurementKeys` only applies to source `FieldSource.PropertyId`. If more than one key is specified, they are matched as exclusive-or. A key has to be a non-null value.
 | Label | string | Optional | null | Friendly name for the field. Certain tokens have special meaning: one of these, {IdentifyingValue}, is included in the suggested labels of AvailableFieldSets. Tokens that do not resolve are "" (empty string). The label is required in a field for all usages except in an identifying field. Label is trimmed of whitespace when used to identify field mappings. 
+| IncludeUom *(Coming Soon)*| bool | Optional | false | Specifies whether to include this field's unit of measure as an additional field mapping in the resolved data view.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Quick_Start_Define_a_Data_View.md:103]: sure

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
+---
+uid: DataViewsUnitsOfMeasure
+---
+
+# Display units of measure (coming soon)
+
+*Note: This topic covers features that are not yet generally available. If you are interested in trialing these pre-release features, contact your account team for more details.*
+
+Data views can be configured to show the units of measure (Uom) from the underlying data source. This topic describes how units of measure is exposed in the resolved data view, as well as the different options to include them in the data set itself.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:9]: sure

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+
+## Units of measure in resolved data view
+In the [resolved data items](xref:DataViewsDataItemsandGroups), each data item field show the unit of measure from the underlying data source. In addition, each data mapping in the [resolved field mapping](xref:DataViewsFieldMappings) show the unit of measure. If a unit of mreasure is not present in the underying data source, the `Uom` property is null.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:13]: changed all 3

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+If the unit of measure is consistent across all groups, then that unit of measure is reported on the [field mapping](xref:DataViewsFieldMappings) itself. Otherwise, the field mapping's `Uom` property is null.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:15]: sure

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md (hunk 1 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:24]: fixed both

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md (hunk 2 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:27]: changed

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md (hunk 3 above)
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:69]: changed

**Comment by d-ng** [comment]
@osimarkrogers all comments addressed, please take a look. Thanks!

**Review by osimarkrogers — APPROVED** [review]

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+# Display units of measure (coming soon)
+
+*Note: This topic covers features that are not yet generally available. If you are interested in trialing these pre-release features, contact your account team for more details.*
+
+Data views can be configured to show the units of measure (Uom) from the underlying data source. This topic describes how to expose units of measure in the resolved data view, as well as the different options to include them in the data set itself.
+
+
+## Units of measure in resolved data view
+In the [resolved data items](xref:DataViewsDataItemsandGroups), each data item field shows the unit of measure from the underlying data source. In addition, each data mapping in the [resolved field mapping](xref:DataViewsFieldMappings) shows the unit of measure. If a unit of measure is not present in the underying data source, the `Uom` property is null.
```
[the remark this answers is not in the document]
Comment (jamesbperry) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:13]: Something like "Units of measure are shown on each data item field, as well as any corresponding data mappings where that data item field is used in the data view."?

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
+---
+uid: DataViewsUnitsOfMeasure
+---
+
+# Display units of measure (coming soon)
+
+*Note: This topic covers features that are not yet generally available. If you are interested in trialing these pre-release features, contact your account team for more details.*
+
+Data views can be configured to show the units of measure (Uom) from the underlying data source. This topic describes how to expose units of measure in the resolved data view, as well as the different options to include them in the data set itself.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:9]: fix in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:13]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+## Units of measure in resolved data view
+In the [resolved data items](xref:DataViewsDataItemsandGroups), each data item field shows the unit of measure from the underlying data source. In addition, each data mapping in the [resolved field mapping](xref:DataViewsFieldMappings) shows the unit of measure. If a unit of measure is not present in the underying data source, the `Uom` property is null.
+
+If the unit of measure is consistent across all groups, then that unit of measure is reported on the [field mapping](xref:DataViewsFieldMappings) itself. Otherwise, the `Uom` property of the field mapping is null.
+
+Note that the [ids of the units of measure](xref:unitsOfMeasure) are shown in resolved resources.
+
+
+## Units of measure in data view
+Data views can be optionally configured to show units of measure within the data set or in labels. The [abbreviations of the units of measure](xref:unitsOfMeasure) are shown in the data view.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:21]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+| 2020-11-06T08:00:00Z | 1.21  | kW        |
+
+The resulting data view in narrow mode:
+| Timestamp            | Field | Value | Uom |
+|----------------------|-------|-------|-----|
+| 2020-11-06T08:00:00Z | Value | 1.21  | kW  |
+
+
+### Show units of measure in field labels
+The field label token {Uom} can be used to add field-level unit of measure to field labels. {Uom} is included in the default field label. If the unit of measure is consistent across all groups, {Uom} is resolved to the field-level unit of measure and displayed in the field label.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:66]: fixed in next update

File: Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+
+Data views can be configured to show the units of measure (Uom) from the underlying data source. This topic describes how to expose units of measure in the resolved data view, as well as the different options to include them in the data set itself.
+
+
+## Units of measure in resolved data view
+In the [resolved data items](xref:DataViewsDataItemsandGroups), each data item field shows the unit of measure from the underlying data source. In addition, each data mapping in the [resolved field mapping](xref:DataViewsFieldMappings) shows the unit of measure. If a unit of measure is not present in the underying data source, the `Uom` property is null.
+
+If the unit of measure is consistent across all groups, then that unit of measure is reported on the [field mapping](xref:DataViewsFieldMappings) itself. Otherwise, the `Uom` property of the field mapping is null.
+
+Note that the [ids of the units of measure](xref:unitsOfMeasure) are shown in resolved resources.
```
[the remark this answers is not in the document]
Comment (d-ng) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md:17]: fixed in next update.

I'm not following what you mean by "formatting seems to be hiding it". can you elaborate?

[the remark this answers is not in the document]
Comment (jamesbperry) [reply Content_Portal/Documentation/DataViews/DefineaDataView/Units_of_Measure.md]: Looks good! The "formatting ... hiding" thing was just that on my screen, the suggestion text appears black, and doesn't show the cross reference hyperlink.

**Review by jamesbperry — APPROVED** [review]

**Pull request merged**